db/silkworm/etl/collector.hpp

```diff
@@ -0,0 +1,54 @@
+/*
+   Copyright 2020 The Silkworm Authors
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+#ifndef SILKWORM_ETL_COLLECTOR_H
+#define SILKWORM_ETL_COLLECTOR_H
+
+#include <silkworm/db/chaindb.hpp>
+#include <silkworm/etl/error.hpp>
+#include <silkworm/etl/buffer.hpp>
+#include <silkworm/etl/file_provider.hpp>
+
+namespace silkworm::etl {
```

Comment:
Let's add a comment somewhere that ETL stands for Extract, Transform, Load:
https://en.wikipedia.org/wiki/Extract,_transform,_load